docs/api/math/isna.rst

```diff
@@ -1,8 +1,47 @@
 
 .. xfunction:: datatable.math.isna
-    :src: src/core/expr/funary/floating.cc resolve_op_isna
+    :src: src/core/expr/funary/fexpr_isna.cc pyfn_isna
     :tests: tests/math/test-isna.py
     :cvar: doc_math_isna
-    :signature: isna(x)
+    :signature: isna(cols)
+
+    Tests if the column elements are missing values. 
+
+    Parameters
+    ----------
+    cols: FExpr
+        Input columns.
+
+    return: FExpr
+        f-expression that returns `0` for valid elements and `1` otherwise. All the resulting columns will have `bool8` stypes and as many rows/columns as there are in `cols`.
```

Comment:
This line needs to be split, otherwise it doesn't fit in some terminals. We usually using 80 chars per line in datatable.